docs/index.md

```diff
@@ -0,0 +1,24 @@
+Hyperbahn
+=========
+
+Hyperbahn is a service discovery and routing network that allows simple, reliable
+communication with any service without any knowledge of where that service runs.
+
+Hyperbahn supports the TChannel RPC protocol and works with Ringpop's gossip protocol
+and consistent hash ring topology.
+
+Hyperbahn and TChannel clients are currently supported in Go, Python, and Node.js.
+
+* Go
+    * [Guide](go-guide.md)
+    * [API Documentation](http://godoc.org/github.com/uber/hyperbahn-go)
+
+
+* Node
+    * [Guide](http://hyperbahn-node.readthedocs.org/en/latest/GUIDE/)
+    * [API Documentation](http://hyperbahn-node.readthedocs.org/en/latest/)
```

Comment:
@Lupie Currently the documentation for the "hyperbahn client" that developers use lives in tchannel. tchannel ships with thrift & hyperbahn integration so any documentation for "how do i talk to hyperbahn" lives in tchannel.

Any documentation for "how do I run hyperbahn locally" lives here.

There's definitely a bunch of overlap here that can be improved, but not quite sure what the best seperation is.